multiview/construct/rsm.py

```diff
@@ -46,25 +49,26 @@ def random_subspace_method(X, n_features=None, n_views=1):
     # check if n_feaures is between 0 and 1
     if n_features < 1:
         n_features = int(n_features*cols)
-        
+
     views = []
-    
+
     for _ in range(n_views):
         view = np.copy(X)
         features_selected = []
-        
+
         while len(features_selected) != n_features:
-            
+
             feature = random.randint(0, cols - 1)
```

Comment:
Should there be a parameter to allow the user to choose the seed for this function?

Reply:
I think it sufficient to set seed before calling the function